test/e2e/containers_conf_test.go

```diff
@@ -311,7 +311,7 @@ var _ = Describe("Podman run", func() {
 		session = podmanTest.Podman([]string{"run", ALPINE, "date", "+'%H %Z'"})
 		session.WaitWithDefaultTimeout()
 		Expect(session.ExitCode()).To(Equal(0))
-		Expect(session.OutputToString()).To(ContainSubstring("EST"))
+		Expect(session.OutputToString()).To(ContainSubstring("EDT"))
```

Comment:
Won't this change/fail in the fall when we go back to EST?  Seems like a check that needs to be modified....

Reply:
DO NOT MERGE THIS. Use 8de560703 instead.

Reply:
Done, thanks!